astroquery/mast/cutouts.py

```diff
@@ -117,36 +120,64 @@ def get_sectors(self, coordinates=None, radius=0*u.deg, objectname=None):
-            One and only one of coordinates and objectname must be supplied.
+            One and only one of coordinates, objectname, and moving_target must be supplied.
         radius : str, float, or `~astropy.units.Quantity` object, optional
             Default 0 degrees.
             If supplied as a float degrees is the assumed unit.
             The string must be parsable by `~astropy.coordinates.Angle`. The
             appropriate `~astropy.units.Quantity` object from
             `astropy.units` may also be used.
+            This argument is ignored if moving_target is supplied.
         objectname : str, optional
             The target around which to search, by name (objectname="M104")
             or TIC ID (objectname="TIC 141914082").
-            One and only one of coordinates and objectname must be supplied.
+            One and only one of coordinates, objectname, moving_target must be supplied.
+        moving_target : str, optional
+            The name or ID (as understood by the
+            `JPL ephemerides service <https://ssd.jpl.nasa.gov/horizons.cgi>`__)
+            of a moving target such as an asteroid or comet.
+            One and only one of coordinates, objectname, and moving_target must be supplied.
+        mt_type : str, optional
+            The moving target type, valid inputs are majorbody and smallbody. If not supplied
+            first majorbody is tried and then smallbody if a matching majorbody is not found.
+            This argument is ignored unless moving_target is supplied.
 
         Returns
         -------
         response : `~astropy.table.Table`
-            Sector/camera/chip information for given coordinates/raduis.
+            Sector/camera/chip information for given coordinates/objectname/moving_target.
         """
 
-        # Get Skycoord object for coordinates/object
-        coordinates = parse_input_location(coordinates, objectname)
+        if moving_target:
 
-        # If radius is just a number we assume degrees
-        radius = Angle(radius, u.deg)
+            # Check only ony object designator has been passed in
+            if objectname or coordinates:
+                raise InvalidQueryError("Only one of objectname, coordinates, and moving_target may be specified.")
 
-        params = {"ra": coordinates.ra.deg,
-                  "dec": coordinates.dec.deg,
-                  "radius": radius.deg}
+            params = {"obj_id": moving_target}
 
-        response = self._service_api_connection.service_request_async("sector", params)
-        response.raise_for_status()  # Raise any errors
+            # Add optional parameter is present
+            if mt_type:
+                params["obj_type"] = mt_type
+
+            response = self._service_api_connection.service_request_async("mt_sector", params)
+
+        else:
+
+            # Get Skycoord object for coordinates/object
+            coordinates = parse_input_location(coordinates, objectname)
```

Comment:
I see that this has been the same before, but maybe it's worth using a different name here and below in the other methods, too that `coordinates` that is also a non-parsed, and maybe None input kwarg.

Reply:
We rec would be to file and issue and have this be a seperate PR, bc this is the convention throughout the MAST module, so I think it would be clearer if they all got changed at once.